### ISSUE TITLE
container select mode bug on editor

### ISSUE THREAD

**Issue by kingpirux** [Scirra/Construct-3-bugs#3656]
## Problem description
if the picked option of container is "all" but there are different type of elements, the menu options became inaccesible for all the elements in the container

## Attach a .c3p

https://drive.google.com/file/d/1mph1zcrumi1BBDtoXl2k4PHmxs1U-jSX/view?usp=sharing

## Steps to reproduce

1. open and select any element
2. change the select mode to "all"
3. select any element

## Observed result

all the options disapear

## Expected result

to see the normal propierties of the selected element or at least the option to change select mode again

## More details

**Affected browsers/platforms:** 

**First affected release:** 

## System details

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by AshleyScirra** [comment]
This is by design. It's not related to containers: select both objects with no container created, and you get the same properties. It's the subset of properties that both types of objects have (which isn't many as they are different plugin types). The container selection mode "all" forces both to always be selected, unless you either use Ctrl+select to unselect one again, or hold Alt when selecting an instance to bypass container selection. I noticed the Alt+Select shortcut wasn't documented in the keyboard shortcuts manual entry, so I added that.
